metabase/metabase
# [WIP - open to let the e2e run] better error for expired token in dashcard request

Closes https://github.com/metabase/metabase/issues/11728

### Description

`/embed/*` endpoints don't return error codes or too much information about the error by choice (see [here](https://github.com/metabase/metabase/blob/0a6e5ad9e9419cd1a2d525108bd7136750309316/src/metabase/api/routes/common.clj#L13-L16))
Nonetheless the linked issue needed a better explanation to not leave the user confused.
The issue arises when you have a dashboard with filters in a static embed and the user changes the filter after the signed JWT has expired.

### How to verify

- Create a dashboard
- Add a filter and connect it to a question
- Do a static embed of the dashboard, set the jwt to expire at the time of the creation (there is a leeway of 60 seconds anyway, so you'll have time to open it)
- load the dashboard, it should load and work just fine
- wait for the minute to pass so that the jwt expires
- change the value of the filter, it will trigger a reload of the card that will run into the error
- it should display the new error prompting to reload the page

### Demo

<img width="1802" alt="image" src="https://github.com/metabase/metabase/assets/1914270/a9e1f7cc-24e7-46cf-84e0-4419a9ae8e2e">

### Checklist
